scripts/systemd/xcash-rpc-wallet.service

```diff
@@ -1,11 +1,13 @@
 [Unit]
 Description=X-Cash RPC wallet background process
+After=network.target xcash-daemon.service
  
 [Service]
 Type=simple
 User=root
+ExecStartPre=/usr/bin/sleep 10
```

Comment:
Is there a way we can not include this, since If someone needs to start them manually without the script, it would cause a 10 second delay? can we move the 10 second delay to inside the script instead?

Reply:
Yup, but if you enable all the services to run with the server you need to do it before the process start and this is why I had to do this way. Another aproach would be to make an internal "retry every X seconds" until it connects to the daemon

Reply:
ah okay, let me think about this one then

Reply:
@zachhildreth I think we can do it letting it just break and restart it automatically. 
Will break the first time if the daemon dont start quickly but the next round will do his job, for example 

We remove: 
```
ExecStartPre=/usr/bin/sleep 10
```
And add 
```
Restart=on-failure
RestartSec=5
```
Here seems the rpc wallet wait for the daemon without any issue, and if it can't reach it internally seems to retry it